source/nordic_sdk/components/libraries/experimental_section_vars/section_vars.h

```diff
@@ -59,7 +59,7 @@
  *
  * @param[in]   section_name    Name of the section to register
  **/
-#if defined __CC_ARM
+#if defined (_CC_ARM) || (__ARMCC_VERSION)
```

Comment:
This macro check has changed from `__CC_ARM` to `_CC_ARM`, was that on purpose?

Reply:
Mistake, plus I've put that define in the wrong place